hw/ip/uart/data/uart.hjson

```diff
@@ -3,9 +3,10 @@
 // SPDX-License-Identifier: Apache-2.0
 {
   name:               "uart",
+  human_name:         "UART",
   one_line_desc:      "Full duplex serial communication interface, supports bit rates of up to 1 Mbit/s",
   one_paragraph_desc: '''
-  The Universal Asynchronous Receiver/Transmitter (UART) hardware block provides industry standard serial communication with external devices via two wires at a programmable baud rate.
+  The Universal Asynchronous Receiver/Transmitter (UART) provides industry standard serial communication with external devices via two wires at a programmable baud rate.
```

Comment:
Suggest removing bracket human_name as this is not done for most other blocks
e.g.
`The Universal Asynchronous Receiver/Transmitter provides industry standard serial communication with external devices via two wires at a programmable baud rate.`